src/benchmarks/micro/corefx/System.Text.Json/Utf8JsonWriter/ArrayBufferWriter.cs

```diff
@@ -0,0 +1,136 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers;
+using System.Diagnostics;
+
+namespace System.Text.Json
+{
+    internal sealed class ArrayBufferWriter<T> : IBufferWriter<T>
+    {
+        private T[] _buffer;
+        private int _index;
+
+        private const int MinimumBufferSize = 256;
+
+        public ArrayBufferWriter()
+        {
+            _buffer = new T[MinimumBufferSize];
+            _index = 0;
+        }
+
+        public ArrayBufferWriter(int initialCapacity)
+        {
+            if (initialCapacity <= 0)
+                throw new ArgumentException(nameof(initialCapacity));
+
+            _buffer = new T[initialCapacity];
+            _index = 0;
+        }
+
+        public ReadOnlyMemory<T> WrittenMemory
```

Comment:
Could you please make the properties use the shorthand `=>`? Same below?
```csharp
public ReadOnlyMemory<T> WrittenMemory => _buffer.AsMemory(0, _index);
```

Reply:
Sure. Ideally this file/type will go away completely and be built-in. I generally was avoiding any cosmetic changes to this file.

Reply:
What timescale does it go away on?

Reply:
Likely by preview6.

Reply:
There is still an asterisk/caveat though, since the built-in type will likely be .NET Core 3.0 specific. I'll likely still need a copy for older TFMs.